Mirror/Runtime/NetworkServer.cs

```diff
@@ -889,6 +889,9 @@ internal static void SendSpawnMessage(NetworkIdentity identity, NetworkConnectio
 
             if (LogFilter.Debug) { Debug.Log("Server SendSpawnMessage: name=" + identity.name + " sceneId=" + identity.sceneId + " netid=" + identity.netId); } // for easier debugging
 
+            MsgType messageType;
```

Comment:
can you do the sendspawnmessage improvements in another pull request please?
only add rotation for this one and we'll merge